rpcserver.go

```diff
@@ -723,7 +732,8 @@ func (r *rpcServer) SendPaymentSync(ctx context.Context,
 	if err := r.server.htlcSwitch.SendHTLC(htlcPkt); err != nil {
 		return nil, err
 	}
-
+	payment := constructPayment(path, btcutil.Amount(nextPayment.Amt), rHash[:])
+				r.server.chanDB.AddPayment(payment)
```

Comment:
Looks like this file hasn't properly been run through `go-fmt` as dictated in the contribution guidelines. One easy way to do this is to find a plugin for you editor/IDE that automatically runs `go-fmt` over the file on each save. 

IIRC you like using IntelliJ, so you might want to check out their new golang centered IDE: https://www.jetbrains.com/go/